backend-nest/libs/common/src/profile/employee.profile.ts

```diff
@@ -12,11 +18,51 @@ export class EmployeeProfile extends AutomapperProfile {
 
   override get profile() {
     return (mapper) => {
-      createMap(mapper, Employee, ReadEmployeeDto, forMember((dest) => dest.roles, mapFrom(source => source.roles)));
-      createMap(mapper, Employee, ValidateUserDto, forMember((dest) => dest.roles, mapFrom(source => source.roles)));
-      createMap(mapper, ValidateUserDto, ReadUserDto, forMember((dest) => dest.roles, mapFrom(source => source.roles))); 
-      createMap(mapper, CreateEmployeeDto, Employee, forMember((dest) => dest.roles, mapFrom(source => source.roles)));
-      createMap(mapper, UpdateEmployeeDto, Employee, forMember((dest) => dest.roles, mapFrom(source => source.roles)));
```

Comment:
maybe create a private function for each of this action, like that you can explain what are you mapping